project/api/write.py

```diff
@@ -1,26 +1,22 @@
-import os, sys, json, pdb, random, hashlib, urllib2, pprint, urllib, PIL
-from models import Account, Category, Civi, CiviImage, Hashtag, Activity, Invitation
+import json, urllib, PIL
+from models import Account, Category, Civi, CiviImage, Activity, Invitation
```

Comment:
Go ahead and alphabetize these imports while we are here. It would make it slightly easier to read.

Reply:
I rearranged the import statements, I even moved core system packages up top, then django, then civi packages.